misc/config_definitions.json

```diff
@@ -950,6 +950,14 @@
             ],
             "type": "array"
         },
+        "top_device_types": {
```

Comment:
I don't think this is a worthwhile setting.  I'd prefer it to be removed unless you have a good reason it should be a setting.

Reply:
my thought about adding  a setting is this from you:
"Also, I suggest we limit the number of columns that can be shown by default (say 5)."
You say 5, i prefer to see all, another one prefers to see another count of types by default

Reply:
But, you just enable the columns then right?

Reply:
by default the top n Devicetypes will be shown 
and n is the setting value

Reply:
Sorry to take a hard line on this, but I don't think it is needed.

Adding settings is degenerative for software.
Settings are difficult to remove and make unused/untested code paths very likely.
Whenever adding a setting, it is worth taking extra time to see if you can do without the setting.
One method is to omit the setting then see if anyone even asks for it.

Reply:
You prefer Top 5 (see comment from 26. Januar) , i prefer Top 7, so it's a useful to have a WebGUI Setting for it.
Also, most listed DeviceTypes before this PR, are useless for me. Thats why i created this PR to see the most used Device Types and not Device Types i am not using.
Nevertheless we need this Overview page, to see the most cirtical locations, and what we've there in summary.
So i think it's needed and useful :-)

Reply:
Hmm, shouldn't it be a dashboard widget to show device types at locations?  It would be appropriate to have settings there.
This screen is more for managing which locations exist.

Reply:
Could be, but it's nothing which has to be used such frequently like monitoring Overview stuff like Device State, Application State or similar stuff.
I think it can stay on the place it is, but my PR would be very useful, especially to get an instant overview about all Locations (e.G. Datacenter, company locations, Buildings, ...) for getting an instant tabula rasa with its status and importance and usage.

Reply:
I'm feeling more and more convinced that device types should be completely removed from the location page and a widget should be added for the info.  Perhaps a description field could be added to show on the location overview page instead.

Reply:
you mean to cut out 

- (Device Count)
- all Device Type
- Devices Down

so only Location relevant stuff 
- Location Name
- Geo Location
-  (Device Count)
- Buttons

is remaining on Geo Location Page and recreate current view of this PR as a widget?

Mmh, was not my intention, but i think this could work.

Reply:
Yes, I know but I think it separates concerns a little and makes things better.
Location management page vs location status page.

Thanks for having patience with me @SourceDoctor